index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <title>My First Portfolio</title>
+</head>
+<body>
+    <nav class="mainMenu">
+        <p class ="menuLogo">Ruri Enciso</p>
+        <div class="menuIcon">
+            <div class="menuLvl"></div>
+            <div class="menuLvl"></div>
+            <div class="menuLvl"></div>
+        </div>
+    </nav>
```

Comment:
- [ ] Dear @RuriEnciso, Please wrap the container of this section with a `header` tag instead of a `nav` tag. Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.
> use the `nav` tag to wrap your hamburger button only.